rst2pdf/findfonts.py

```diff
@@ -179,26 +172,25 @@ def findFont(fname):
             return None
     return font
 
-def findTTFont(fname):
 
+def findTTFont(fname):
     def get_family(query):
-        data = subprocess.check_output(["fc-match", query])
+        data = make_string(subprocess.check_output(["fc-match", query]))
         if six.PY2:
-            data = data.decode('UTF-8')
+            data = data.decode("UTF-8")
         for line in data.splitlines():
             line = line.strip()
             if not line:
                 continue
-            fname, family, _, variant = line.split('"')[:4]
-            family = family.replace('"', '')
+            family = line.split('"')[1].replace('"', "")
             if family:
                 return family
         return None
 
     def get_fname(query):
-        data = subprocess.check_output(["fc-match", "-v", query])
+        data = make_string(subprocess.check_output(["fc-match", "-v", query]))
         if six.PY2:
-            data = data.decode('UTF-8')
+            data = data.decode("UTF-8")
```

Comment:
As `make_string` does a utf-8 decode, we don't need this `if six.PY2:` statement I think.

Reply:
agreed